app/models/abuse_report.rb

```diff
@@ -55,15 +55,15 @@ def email_and_send
   end
 
   def send_report
-    return unless %w(staging production).include?(Rails.env)
+    #return unless %w(staging production).include?(Rails.env)
```

Comment:
I think you just commented this out for testing on your machine... we still want it in